presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -4733,6 +4734,39 @@ public void testShowColumnMetadata()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testShowViews()
+    {
+        String viewName = "test_show_views";
+
+        @Language("SQL") String createView = "CREATE VIEW " + viewName + " AS SELECT abs(1) as whatever";
+
+        Session testSession = testSessionBuilder()
+                .setIdentity(Identity.ofUser("test_view_access_owner"))
+                .setCatalog(getSession().getCatalog().get())
+                .setSchema(getSession().getSchema().get())
+                .build();
+
+        assertUpdate(createView);
+        try {
```

Comment:
I would remove this `try-finally` block. View name is unique and no other test should depend that there is no view in the system.